src/fiat/system/ec_get_cycles.c

```diff
@@ -40,6 +40,8 @@ long long int ec_get_cycles_()
 #include <stdint.h>
 #include <sys/platform/ppc.h>
 #include <sched.h>
+#include <stdio.h>
+#include <linux/limits.h>
```

Comment:
@ioanhadade I am not sure if the "linux/limits.h" file is present with AIX on powerpc.

Reply:
There is a limits.h in AIX but wouldnt be under linux. I think AIX in HPC is history.

Reply:
Thanks for your input. Upon request we can always make more fixups.